zcash_client_sqlite/src/wallet.rs

```diff
@@ -1594,6 +1805,9 @@ pub(crate) fn truncate_to_height<P: consensus::Parameters>(
         wdb.with_sapling_tree_mut(|tree| {
             tree.truncate_removing_checkpoint(&block_height).map(|_| ())
         })?;
+        wdb.with_sapling_tree_mut(|tree| {
```

Comment:
Bug: This bugfix just duplicated the Sapling block without adjustment.
```suggestion
        wdb.with_orchard_tree_mut(|tree| {
```

Reply:
Crap, thank you.